aws_lambda_builders/workflows/custom_make/make.py

```diff
@@ -84,6 +84,8 @@ def run(self, args, env=None, cwd=None):
 
         out, err = p.communicate()
 
+        LOG.info(out.decode("utf8").strip())
```

Comment:
Should we also think about printing `err` output as well?

Reply:
Isn't it sufficient that the `err` is raised by the `MakeExecutionError` if the return code is not `0` in lines 89-90?